docs/extending-overseerr/reverse-proxy-examples.md

```diff
@@ -112,7 +112,7 @@ server {
     add_header Referrer-Policy "no-referrer";
     # HTTP Strict Transport Security
     add_header Strict-Transport-Security "max-age=63072000; includeSubDomains" always;
-    # Reduce XSS risks (Content-Security-Policy)
+    # Reduce XSS risks (Content-Security-Policy) - add URLs whenever necessary
     add_header Content-Security-Policy "default-src 'self'; connect-src 'self' https://plex.tv; style-src 'self' 'unsafe-inline' https://rsms.me/inter/inter.css; script-src 'self'; img-src 'self' data: https://plex.tv https://assets.plex.tv https://secure.gravatar.com https://gravatar.com https://i2.wp.com https://image.tmdb.org; font-src 'self' https://rsms.me/inter/font-files/" always;
```

Comment:
I think we can leave it up to date with:
```
add_header Content-Security-Policy "default-src 'self'; connect-src 'self' https://plex.tv; style-src 'self' 'unsafe-inline' https://rsms.me/inter/inter.css; script-src 'self'; img-src 'self' data: https://plex.tv https://assets.plex.tv https://secure.gravatar.com https://gravatar.com https://i2.wp.com https://image.tmdb.org; font-src 'self' https://rsms.me/inter/font-files/" always;
```
Maybe comment out the line so it is not enabled by default? 

Put a note in there such as:
This list is subject to change. If you enable this please add any missing entries.

Reply:
I had updated the comment to include "add URLs whenever necessary" which is effectively what you are suggesting.

I don't think this example should stop being updated because it is intended as a guide and therefore needs to be up to date.


You mentioned:
> every user will have to update their config when this changes

Surely that is a given? Those who are using a reverse proxy would **always** have to update their configuration when a new source is utilised.  Updating the guides helps to communicate that more effectively. Also, new URLs may be a common occurrence now but it should taper off as the project matures.

It raises another question though, why is both https://secure.gravatar.com and https://gravatar.com necessary? Should the local user gravatar be changed to point to https://secure.gravatar.com instead?

Reply:
I authored the LE/NGINX configuration, and using `https://gravatar.com` should be fine. I've been using it this way in both Chromium and Firefox based browsers. It may have been that I originally had only `https://secure.gravatar.com`, which would have been an error on my part.

@hirenshah, what browser were you using when you encountered issue #660?